pmg/views.py

```diff
@@ -234,10 +235,16 @@ def bill(bill_id):
         'introduced': 1,
     }
     history = bill_history(bill)
+
+    if bill['status']:
```

Comment:
To fix TypeError here (because `bill['status']` exists and the value is `None`, you could do 
```python
if bill.get('status', None):
```
which says get status, but if it's not present in the dict, default to None.